reference/configuration/doctrine.rst

```diff
@@ -155,7 +155,13 @@ which is the first one defined or the one configured via the
 ``default_connection`` parameter.
 
 Each connection is also accessible via the ``doctrine.dbal.[name]_connection``
-service where ``[name]`` is the name of the connection.
+service where ``[name]`` is the name of the connection. Or directly in the controller with 
+getConnection method and the name of the connection::
+
+.. code-block:: php
+
+    $connection = $this->getDoctrine()->getConnection('customer');
+    $result = $connection->fetchAll('select * from table');
```

Comment:
SELECT name FROM customers

would fit better IMO

Reply:
it's just an example. I don't know the table's name or any field name.

Reply:
Yes but “Select * ... “ shouldn’t be promoted

Reply:
ok